main.go

```diff
@@ -76,36 +98,18 @@ func main() {
 		metav1.NamespaceNone,
 	}
 
-	// Get a config to talk to the apiserver
-	cfg, err := config.GetConfig()
-	if err != nil {
-		klog.Exit(err.Error())
-	}
-
-	// Create a new Cmd to provide shared dependencies and start components
-	mgr, err := manager.New(cfg, manager.Options{
-		NewCache:                cache.MultiNamespacedCacheBuilder(namespaces),
-		LeaderElection:          true,
-		LeaderElectionID:        "performance-addon-operators",
-		LeaderElectionNamespace: namespace,
-		Namespace:               namespace,
-		MetricsBindAddress:      fmt.Sprintf("%s:%d", metricsHost, metricsPort),
+	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+		NewCache:           cache.MultiNamespacedCacheBuilder(namespaces),
+		Scheme:             scheme,
+		MetricsBindAddress: metricsAddr,
+		Port:               9443,
```

Comment:
this need to be a constant

Reply:
Auto generated,  it should have been changed by a patch, will fix.

Reply:
done